### ISSUE TITLE
 Failed reading metadata

### ISSUE THREAD

**Issue by DatrixTHLK** [elix-x/Excore#4]
I wasn't sure if I should post this here or FEI
https://gist.github.com/DatrixTHLK/453a1e980469c3e66712d9fa8d3e1846
Minecraft:1.10.2
Forge: 2023
elix_x: 1.6.2

**Comment by Elix-x** [comment]
Yeah, yeah... Known... Is/will be fixed in 2.0.

_Also, i didn't think that I am versionned..._

**Comment by DatrixTHLK** [comment]
Thanks...

**Comment by drakray** [comment]
Will this be fixed for 1.10.2?

**Comment by Elix-x** [comment]
@drakray Sadly, no.

**Comment by drakray** [comment]
@Elix-x ok, thanks

**Comment by Elix-x** [comment]
Woops.

**Comment by bookerthegeek** [comment]
Still an issue.
```
[03:19:08] [Client thread/WARN]: Failed reading metadata of: http://www:http://i.imgur.com/qPvG90n.png
com.google.gson.JsonSyntaxException: com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 2 column 2
	at com.google.gson.JsonParser.parse(JsonParser.java:65) ~[JsonParser.class:?]
	at net.minecraft.client.resources.SimpleResource.func_110526_a(SourceFile:65) ~[bxj.class:?]
	at net.minecraft.client.renderer.texture.SimpleTexture.func_110551_a(SourceFile:36) [bwa.class:?]
	at net.minecraft.client.renderer.texture.TextureManager.func_110579_a(TextureManager.java:67) [bwf.class:?]
	at code.elix_x.excore.client.thingy.ThingyDisplay.cacheIcons(ThingyDisplay.java:82) [ThingyDisplay.class:?]
	at code.elix_x.excore.client.thingy.Thingy$1.run(Thingy.java:33) [Thingy$1.class:?]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) [?:1.8.0_121]
	at java.util.concurrent.FutureTask.run(Unknown Source) [?:1.8.0_121]
	at net.minecraft.util.Util.func_181617_a(SourceFile:45) [h.class:?]
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1045) [bcx.class:?]
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:371) [bcx.class:?]
	at net.minecraft.client.main.Main.main(SourceFile:124) [Main.class:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_121]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_121]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_121]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_121]
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135) [launchwrapper-1.12.jar:?]
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28) [launchwrapper-1.12.jar:?]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_121]
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_121]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source) ~[?:1.8.0_121]
	at java.lang.reflect.Method.invoke(Unknown Source) ~[?:1.8.0_121]
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:236) [NewLaunch.jar:?]
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:297) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.listen(EntryPoint.java:162) [NewLaunch.jar:?]
	at org.multimc.EntryPoint.main(EntryPoint.java:53) [NewLaunch.jar:?]
Caused by: com.google.gson.stream.MalformedJsonException: Use JsonReader.setLenient(true) to accept malformed JSON at line 2 column 2
	at com.google.gson.stream.JsonReader.syntaxError(JsonReader.java:1505) ~[JsonReader.class:?]
	at com.google.gson.stream.JsonReader.checkLenient(JsonReader.java:1386) ~[JsonReader.class:?]
	at com.google.gson.stream.JsonReader.doPeek(JsonReader.java:531) ~[JsonReader.class:?]
	at com.google.gson.stream.JsonReader.peek(JsonReader.java:414) ~[JsonReader.class:?]
	at com.google.gson.JsonParser.parse(JsonParser.java:60) ~[JsonParser.class:?]
	... 25 more
```
Although cool picture. ;)
![Troll Face](http://i.imgur.com/qPvG90n.png)

**Comment by Elix-x** [comment]
EXCore-2.0.0-beta1-1.11.2?

**Comment by bookerthegeek** [comment]
Latest for 1.10.2
- ~~ElecCore-1.10.2-1.7.409~~

**Comment by Elix-x** [comment]
First off, not ElecCore and not 1.7.409 (this is not my mod).
Second, sadly i can't fix it in 1.10.2. But this is fixed in 1.11.2 version.

**Comment by bookerthegeek** [comment]
Derp... My bad. I grabbed the wrong mod. 

- Excore-1.6.2-1.10.2

**Comment by Elix-x** [comment]
Still, you're playing in 1.10.2 and sadly i can't fix it there. This bug is fixed in 1.11.2 though.

**Comment by bookerthegeek** [comment]
Understood. Looking forward to when all of the mods in my pack are on the same MC version.
